semcore/color-picker/src/style/color-picker.shadow.css

```diff
@@ -0,0 +1,166 @@
+@import '@semcore/utils/style/var.css';
+
+SColorPickerPopper {
+  max-width: 234px;
```

Comment:
I thought that more than 234px will be too large for color picker. and why exactly 234px - because I decided 6 items in row will be maximum and I have padding 16px, and between color items should be some gap, that's why I tried different width sizes :D 
but if you have better variant I will be glad to hear it